### ISSUE TITLE
Checking FB authorization when cache is empty

### ISSUE THREAD

**Issue by VinceOPS** [Telerik-Verified-Plugins/Facebook#19]
Hi!

On iOs (9.3+ currently, but I assume the "issue" exists for other versions), when logging-in for the first time, the plugin will open the Facebook app, even if the authorization has already been given.
It results in opening, showing, and quickly closing (automatically) the FB app, which looks really weird.

I already use `getLoginStatus` to check if the user already logged in previously, but it has no effect if the app was recently installed (I guess that's because there is no cache).

```
window.facebookConnectPlugin.getLoginStatus(function(response) {
            if (response.status === 'connected') {
                // ...
            }

            // ...
```

Is there any way to avoid this behavior, by checking if the FB authorization has already been given to the app?

Thanks!

**Comment by EddyVerbruggen** [comment]
I'm going to push a massive upgrade to this plugin tomorrow, essentially pulling in [this repo](https://github.com/jeduan/cordova-plugin-facebook4) and making sure the API doesn't change. Can you test with the new version perhaps?

**Comment by VinceOPS** [comment]
Hi! Sure! I'll let you know!

I assume the issue is quite similar to [this one](http://answers.soom.la/t/solved-how-to-avoid-showing-already-authorized-to-app-dialog-in-android-facebook-sdk/1044/3) on Android.

Btw, this is probably the 20th time I say it, but **Thanks** for all your work for the community :)!!

**Comment by EddyVerbruggen** [comment]
Thanks @VinceOPS, appreciate it!

I just released 1.7.1.2 which uses the Facebook 4 SDK. It's API compatible with the old version so you should be able to drop it in as a replacement.

Let me know if this helps, otherwise reopen and we'll start some serious digging.
